generator/src/test/java/com/scottlogic/deg/generator/walker/ReductiveDecisionTreeWalkerTests.java

```diff
@@ -87,7 +95,7 @@ public void shouldReturnEmptyCollectionOfRowsWhenFirstFieldCannotBeFixed() {
         List<DataBag> result = walker.walk(tree).collect(Collectors.toList());
 
         verify(reductiveFieldSpecBuilder).getDecisionFieldSpecs(eq(rootNode), any());
-        Assert.assertThat(result, empty());
+        assertTrue(result.isEmpty());
```

Comment:
Optional: I would assert equals against an empty list to make the assertion failure message a bit nicer.